packages/svelte-vscode/package.json

```diff
@@ -382,6 +382,16 @@
                     "default": true,
                     "title": "Svelte: Rename",
                     "description": "Enable rename/move Svelte files functionality"
+                },
+                "svelte.plugin.svelte.defaultScriptLanguage": {
```

Comment:
Should we put this inside `completions`? Like `"svelte.plugin.svelte.completions.defaultScriptLanguage"`? @jasonlyu123 what do you think?

Reply:
Hmm. It is also being used in the code action. if we want to further organize it. Maybe the name needs to be more generic. Maybe something like `suggest` or `preferences`?

Reply:
Oh right, it's also part of the code action. In that case, never mind 😄 My fear was that people would mistake this as something like "if I set this I don't need to write `<script lang="ts">` at all" (like the deprecated and discouraged default language of svelte preprocess).